content/en/docs/contribute/localization.md

```diff
@@ -17,18 +17,21 @@ card:
 This page shows you how to [localize](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/) the docs for a different language.
 
 
-
 <!-- body -->
 
-## Getting started
+## Contribute to an existing localization
 
-Because contributors can't approve their own pull requests, you need at least two contributors to begin a localization.
+You can help add or improve content to an existing localization. In [Kubernetes Slack](https://slack.k8s.io/) you'll find a channel speicific the localization. There is also a general [SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations) where you can say hello.
```

Comment:
FixUp, spelling:
In [Kubernetes Slack](https://slack.k8s.io/) you'll find a channel for each localization.